test/tests/api/v1_1/os_install_tests.py

```diff
@@ -1,27 +1,162 @@
+        graph_name = 'Graph.InstallCentOS'
+        os_repo = os.getenv('RACKHD_CENTOS_REPO_PATH', \
+            self.__base + '/repo/centos/{0}'.format(version))
+        body = options
+        if body == None:
+            body = {
+                'options': {
+                    'defaults': {
+                        'kvm': 'undefined', 
+                        'version':version,
+                        'repo': os_repo
+                    }
+                }
+            } 
+        self.__post_workflow(graph_name, nodes, body)
+        
+    def install_esxi(self, version, nodes=[], options=None):
+        graph_name = 'Graph.InstallEsx'
+        os_repo = os.getenv('RACKHD_ESXI_REPO_PATH', \
+            self.__base + '/repo/esxi/{0}'.format(version))
+        body = options
+        if body == None:
+            body = {
+                'options': {
+                    'defaults': {
+                        'installDisk': 'firstdisk',
+                        'version': version, 
+                        'repo': os_repo
+                    }
+                }
+            }
+        self.__post_workflow(graph_name, nodes, body)  
+        
+    def install_suse(self, version, nodes=[], options=None):
+        graph_name = 'Graph.InstallSUSE'
+        os_repo = os.getenv('RACKHD_SUSE_REPO_PATH', \
+            self.__base + '/repo/suse/{0}/'.format(version))
+        body = options
+        if body == None:
+            body = {
+                'options': {
+                    'defaults': {
+                        'kvm': 'undefined', 
+                        'version':version,
+                        'repo': os_repo
+                    }
+                }
+            }
+        self.__post_workflow(graph_name, nodes, body)
+        
+    def install_ubuntu(self, version, nodes=[], options=None):
+        graph_name = 'Graph.InstallUbuntu'
+        os_repo = os.getenv('RACKHD_UBUNTU_REPO_PATH', \
+            self.__base + '/repo/ubuntu')
+        body = options
+        if body == None:
+            body = {
+                'options': {
+                    'defaults': {
+                        'kvm': 'undefined', 
+                        'version':version,
+                        'repo': os_repo
+                    }
+                }
+            }
+        self.__post_workflow(graph_name, nodes, body)
+
+    @test(groups=['centos-6-5-install.v1.1.test'])
+    def test_install_centos_6(self):
+        """ Testing CentOS 6.5 Installer Workflow """
+        self.install_centos('6.5')
+        
+    @test(groups=['centos-7-install.v1.1.test'])
+    def test_install_centos_7(self, nodes=[], options=None):
+        """ Testing CentOS 7 Installer Workflow """
+        self.install_centos('7.0')
+
+    @test(groups=['ubuntu-install.v1.1.test'])
+    def test_install_ubuntu(self, nodes=[], options=None):
+        """ Testing Ubuntu 14.04 Installer Workflow """
+        self.install_ubuntu('14.04')
```

Comment:
@jlongever  I'm afraid '14.04' as a `version` will not works. Because the `version` param will be render in `ubuntu_preseed`, and it's require the release name 'trusty' instead of '14.04'